### DISCUSSION TITLE
gradients stops

### DISCUSSION THREAD

**Discussion by 18121A05L2** [tailwindlabs/tailwindcss#9348] [Help] [answered]
I had a gradient like `background: linear-gradient(130.06deg, #ACA4FE 0%, #5C55AB 35.33%, #2B275A 75.68%);`  , in this we have color stops for each color , how to add that **color percentage stops** and also can we able to add custom like **bg-gradient-[130deg]**

**Comment by wongjn** [comment, marked as answer]
You cannot replicate this background gradient using the existing utility classes, you could instead try using an arbitrary value: `bg-[linear-gradient(130.06deg,#ACA4FE_0%,#5C55AB_35.33%,#2B275A_75.68%)]`.

**Reply by 18121A05L2** [reply]
Thank You brother , i havent know we can use gradient like this , actually i havent found this custom gradient in tailwind docs too
